exercises/static/exercises/follow_line/web-template/assets/controller.js

```diff
@@ -1,35 +1,33 @@
 // Main message controller for websockets
-var running = true;
+var running = false;
 
 // Function to execute the student solution
 function start(){
+    // Manager Websocket
+    if (running == false) {
+        resumeSimulation();
+    }
+
     // Code Websocket
     submitCode();
 
     // GUI Websocket
     unpause_lap();
-
-    // Mark the variables
-    running = true;
 }
 
 // Function to stop the student solution
 function stop(){
-    // Code Websocket
-    stopCode();
+    // Manager Websocket
+    if (running == true) {
+        stopSimulation();
+    }
 
     // GUI Websocket
     pause_lap();
-
-    // Mark the variables
-    running = false;
 }
 
 // Function to reset the simulation
 function reset(){
-    // Code websocket
-    stopCode();
```

Comment:
Not sure about this, but does the reset work fine after removing this?

There are 2 test cases for reset:
- Pause and then reset
- Reset a running code(the car will reset it's position and start running the same code)

Does the Lap Time and everything else work fine without this?